server/model/opening-hours.js

```diff
@@ -18,60 +18,59 @@ type OpeningHoursDay = {|
 export type OpeningHours = Array<OpeningHoursDay>;
 
 export const galleryOpeningHours: OpeningHours = [
-  {dayOfWeek: 'Monday',    closes: '00:00', note: 'Galleries closed'},
-  {dayOfWeek: 'Tuesday',   opens: '10:00', closes: '18:00'},
+  {dayOfWeek: 'Monday', closes: '00:00', note: 'Galleries closed'},
```

Comment:
Exceptions are apparently configurable – http://eslint.org/docs/rules/no-multi-spaces#exceptions – I've not managed to set them up though. Have just turned the rule off for now.